README.md

```diff
@@ -1,11 +1,51 @@
-# Xliff-tasks
-Set of tasks and targets to generate xliff files.
+# xliff-tasks
+
+A set of MSBuild tasks and targets to automatically generate xliff (.xlf) files for localizable resources, and to build satellite assemblies from those xliff files.
+
+## Build Status
 
-Build Status
-------------
 |Windows x64 |
 |:------:|
 |[![][win-x64-build-badge]][win-x64-build]|
+
+## Installing
+
+If you're using the [Roslyn Repo Toolset][roslyn-repo-toolset] then the `XliffTasks` package is already pulled in, and enabled by default.
+
+Otherwise, you'll need to add the dotnet-core feed on MyGet (`https://dotnet.myget.org/F/dotnet-core/api/v3/index.json`) to your nuget.config file, and then add a `PackageReference` for the XliffTasks package, like so:
+
+```
+<PackageReference Include="XliffTasks" Version="0.2.0-beta-000081" />
```

Comment:
Do I need a `PrivateAssets="all"` here? If so, why?

Reply:
Yes, because otherwise, `dotnet pack` or `msbuild /t:Pack` will make a .nupkg that says that your .nupkg depends on xliff-tasks.